frontends/web/src/routes/device/bitbox02/settings.tsx

```diff
@@ -150,6 +151,9 @@ class Settings extends Component<Props, State> {
                                                 mnemonicPassphraseEnabled={deviceInfo.mnemonicPassphraseEnabled}
                                                 getInfo={this.getInfo} />
                                         </div>
+                                        <div className="box slim divide">
```

Comment:
Add it to the same box as the MnemonicPssphraseButton, see also https://github.com/digitalbitbox/bitbox-wallet-app/commit/a70b736ce7ee69a10604c6638d868b5fc9c52579

Reply:
Done!